src/store/modules/web3.ts

```diff
@@ -82,9 +82,9 @@ const actions = {
       commit('HANDLE_CHAIN_CHANGED', network.chainId);
       const account = accounts.length > 0 ? accounts[0] : null;
       let name = '';
-      let profiles: any = [];
+      let profilesData: any = {};
       try {
-        [name, { profiles }] = await Promise.all([
+        [name, profilesData] = await Promise.all([
           getProvider('1').lookupAddress(account),
```

Comment:
We don't need this anymore because we are requesting the ENS name from the `getProfiles` method already. There should be a single request without Promise.all